assets/js/app/Panel.js

```diff
@@ -66,50 +49,107 @@ LocusZoom.Panel = function() {
     this.getBaseId = function(){
         return this.parent.id + "." + this.id;
     };
+
+    // Initialize the layout (should this happen in initialize()?)
+    this.initializeLayout();
     
     return this;
     
 };
 
+LocusZoom.Panel.prototype.initializeLayout = function(){
+
+    // Set panel dimensions, origin, and margin or fall back to default values
+    this.layout.width      = this.layout.width      || 0;
```

Comment:
Yeah, here it's not pulling from DefaultLayout as the default if unspecified like in the other case (Instance.js line 47). I guess we just need to decide which pattern we want to use..